Dockerfile

```diff
@@ -12,19 +12,10 @@ LABEL org.opencontainers.image.ref.name="jnovack/autossh" \
       org.opencontainers.image.revision=$COMMIT \
```

Comment:
Add one line, this is my fault.  I can do it, but then you'd have to rebase, easier if you do it.

```
    org.opencontainers.image.version=$VERSION \
```